### ISSUE TITLE
Error when running gradio

### ISSUE THREAD

**Issue by PutraCoud** [acheong08/Diffusion-ColabUI#124]
content/stable-diffusion-webui
usage:  [-h] [--config CONFIG] [--medvram] [--lowvram] [--ckpt CKPT] [--vae VAE] [--data-dir DATA_DIR] [--models-dir MODELS_DIR] [--allow-code] [--share] [--insecure] [--use-cpu USE_CPU [USE_CPU ...]]
        [--listen] [--port PORT] [--freeze] [--auth AUTH] [--authfile AUTHFILE] [--autolaunch] [--api-auth API_AUTH] [--api-log] [--device-id DEVICE_ID] [--cors-origins CORS_ORIGINS]
        [--cors-regex CORS_REGEX] [--tls-keyfile TLS_KEYFILE] [--tls-certfile TLS_CERTFILE] [--tls-selfsign] [--server-name SERVER_NAME] [--no-hashing] [--no-download] [--profile] [--disable-queue]
        [--debug] [--reset] [--upgrade] [--use-ipex] [--use-directml] [--skip-update] [--skip-requirements] [--skip-extensions] [--skip-git] [--experimental] [--test] [--version] [--ignore]
: error: unrecognized arguments: --gradio-auth webui:123

**Comment by RayZ3R0** [comment]
same problem. bump

**Comment by acheong08** [comment]
Gradio auth has been removed since they have improved security and it is no longer a concern. 

I just checked https://colab.research.google.com/github/acheong08/Diffusion-ColabUI/blob/main/Diffusion_WebUI.ipynb and I have removed auth since that feature got removed from the UI

**Comment by PutraCoud** [comment]
CivitAi removed ? i tried to install in extension menu but it keep saying error for a reason

**Comment by acheong08** [comment]
> CivitAi removed ? i tried to install in extension menu but it keep saying error for a reason

Just restored

**Comment by PutraCoud** [comment]
how ? still not in there

**Comment by acheong08** [comment]
Hmm strange. It should be cloning to the extensions directory.

**Comment by RayZ3R0** [comment]
Yeah colab keeps crashing for me as well, after generating one or two images, it crashes
